tests/PhpcsScanScanCommitTest.php

```diff
@@ -896,4 +924,98 @@ public function testDoScanTest5() {
 			$issues_stats
 		);
 	}
+
+	public function testLintWontScanNumberOfLinesInvalid() {
+		$options_test = vipgoci_unittests_options_test(
+			$this->options,
+			array( 'phpcs-runtime-set', 'github-token', 'token' ),
+			$this
+		);
+
+		if ( -1 === $options_test ) {
+			return;
+		}
+
+		$this->options['commit'] =
+			$this->options['commit-test-phpcs-scan-commit-5'];
+
+		$this->options['phpcs-skip-scanning-via-labels-allowed'] =
+			false;
+
+		$issues_submit = array();
+		$issues_stats = array();
+		$issues_skipped = array();
+
+		vipgoci_unittests_output_suppress();
+
+		$prs_implicated = vipgoci_github_prs_implicated(
+			$this->options['repo-owner'],
+			$this->options['repo-name'],
+			$this->options['commit'],
+			$this->options['github-token'],
+			$this->options['branches-ignore']
+		);
+
+
+		foreach( $prs_implicated as $pr_item ) {
+			$issues_stats[
+			$pr_item->number
+			][
+			'error'
+			] = 0;
+
+			$issues_skipped[ $pr_item->number ] = $this->getDefaultSkippedFilesDueIssuesMock();
+		}
+
+		$issues_skipped[39] = array(
```

Comment:
Should this not be set by the functions called by `vipgoci_phpcs_scan_commit()`? Tested running the unit-test without this and it succeeded. Seems like that this code could be removed.